webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const package = require("./package");
+const widgetName = package.widgetName;
+
+const widgetConfig = {
+    entry: `./src/components/${widgetName}Container.ts`,
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: `src/com/mendix/widget/custom/dropdownsort/${widgetName}.js`,
+        libraryTarget: "umd"
+    },
+    resolve: {
+        extensions: [ ".ts", ".js" ],
+        alias: {
+            "tests": path.resolve(__dirname, "./tests")
+        }
+    },
+    module: {
+        rules: [
+            { test: /\.ts$/, use: "ts-loader" },
+            {
+                test: /\.css$/,
+                loader: ExtractTextPlugin.extract({
+                    fallback: "style-loader",
+                    use: "css-loader"
+                })
+            },
+            {
+                test: /\.scss$/,
+                loader: ExtractTextPlugin.extract({
+                    fallback: "style-loader",
+                    use: "css-loader!sass-loader"
+                })
+            }
+        ]
+    },
+    devtool: "source-map",
+    externals: [ "react", "react-dom", /^mxui\/|^mendix\/|^dojo\/|^dijit\// ],
+    plugins: [
+        new CopyWebpackPlugin([
+            { from: "src/**/*.xml" },
+        ], { copyUnmodified: true }),
+        new ExtractTextPlugin({ filename: `./src/com/mendix/widget/custom/dropdownsort/ui/${widgetName}.css` }),
```

Comment:
Same as above, use `packageName`

Reply:
I cant changed this because the web modeler will fail to load css basing on this line

`
export function getPreviewCss() {
    return require("./ui/DropDownSort.scss");
}
`

Reply:
Done

Reply:
Second look, its okay as you indicated

Reply:
Changing the to lowercase works on local host but does not work on the sandbox. so it is not a good idea to do so